config/routes.rb

```diff
@@ -25,6 +25,8 @@
   get "/_health", to: "health#index"
   post "/update", to: "salt#update"
   post "/accept-minion", to: "salt#accept_minion"
+  post "/remove-minion", to: "salt#remove_minion"
+  post "/reject-minion", to: "salt#reject_minion"
```

Comment:
should we put this API endpoints into the `/minions` resource namespace?

Reply:
Maybe we could leave as it is since we left `accept` as that too unless we move accept too.

Reply:
I moved all minion related to the resource namespace